events/blocks/events-form/events-form.js

```diff
@@ -136,6 +146,7 @@ function createButton({ type, label }, bp) {
         button.classList.remove('submitting');
         if (!respJson) return;
 
+        if (respJson.ok) eventFormSendAnalytics(bp, 'Form Submit');
```

Comment:
This is probably confusing on our API's part -- a bad response will contain ok: false, but a good response will just have the rsvpData. I would:
1. move this after L161 and drop the respJson.ok check
2. add a early return after L160

Reply:
@qiyundai, as discussed, rather than making the above updates, I've added the eventFormSendAnalytics call to line 161 and I've removed the  problematic "!respJson.ok" from the existing code.